### PR TITLE
PerformancePaintTiming in Firefox

### PR DESCRIPTION
**Pull request by jakub-g**
Tested with BrowsterStack

Release docs:
https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/84#apis

### CODE REVIEW

File: api/PerformancePaintTiming.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
             "version_added": "≤79"
           },
           "firefox": {
-            "version_added": null
+            "version_added": "84"
           },
           "firefox_android": {
             "version_added": null
```
Comment (ddbeck) [inline api/PerformancePaintTiming.json:20]: Looks like this also appeared in Android

```suggestion
            "version_added": "84"
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
One suggestion to expand on this a little. Thank you, @jakub-g.

**Comment by jakub-g** [comment]
@ddbeck updated :)

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**